app/src/main/java/com/randomappsinc/studentpicker/home/HomeActivity.java

```diff
@@ -76,6 +78,10 @@ public void onSlide(@NonNull View bottomSheet, float slideOffset) {
             showPleaseRateDialog();
         }
 
+        importFromCsv.setText(preferencesManager.isOnFreeVersion() ?
+                getString(R.string.import_from_csv_file_premium) :
```

Comment:
TextView can take in a @StringRes directly, so we can remove the getString() for more compact code.